raitools/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.17.2
+pandas>=0.25.1
+scipy>=1.4.1
+scikit-learn>=0.22.1
+lightgbm>=3.1.1
```

Comment:
Is the minimum lightgbm version needed at 3.1.1? This might be an issue when itegrating with azuremlcli.

Reply:
changed it to lightgbm>=2.2.3